targets/ramips/profiles/usb_mt7620/config

```diff
@@ -1185,38 +1392,38 @@ CONFIG_BUSYBOX_DEFAULT_FEATURE_SH_NOFORK=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_SYSLOGD_READ_BUFFER_SIZE=0
 # CONFIG_BUSYBOX_DEFAULT_FEATURE_IPC_SYSLOG is not set
 CONFIG_BUSYBOX_DEFAULT_FEATURE_IPC_SYSLOG_BUFFER_SIZE=0
-# CONFIG_BUSYBOX_DEFAULT_LOGREAD is not set
-# CONFIG_BUSYBOX_DEFAULT_FEATURE_LOGREAD_REDUCED_LOCKING is not set
 # CONFIG_BUSYBOX_DEFAULT_FEATURE_KMSG_SYSLOG is not set
-# CONFIG_BUSYBOX_DEFAULT_KLOGD is not set
-# CONFIG_BUSYBOX_DEFAULT_FEATURE_KLOGD_KLOGCTL is not set
-CONFIG_BUSYBOX_DEFAULT_LOGGER=y
+CONFIG_PACKAGE_ca-bundle=m
```

Comment:
Without this requirement the package msmtp threw an error to me and didn't build, and you fixed it in ````package/msmtp/Makefile```` by adding it as a dependency: https://www.gargoyle-router.com/phpbb/viewtopic.php?f=8&t=11430&sid=1573575bf75f468070ebd5d68ceab143&start=20#p50216

Or are you saying that I should just give ca-certificates-combined as dependency of msmtp, instead of ca-bundle?

Reply:
This should be resolved now.